src/cladetime/util/reference.py

```diff
@@ -71,7 +71,7 @@ def _get_s3_object_url(bucket_name: str, object_key: str, date: datetime) -> Tup
 
 
 def _run_nextclade_cli(
-    nextclade_cli_version: str, nextclade_command: list[str], output_file: Path, input_files: list[Path] | None = None
+    nextclade_cli_version: str, nextclade_command: list[str], output_path: Path, input_files: list[Path] | None = None
```

Comment:
Chipping away at #52. The `docker run` command only needs to know about the output_path (for volume mounting). The actual file nam efor Nextclade CLI output is specified in `nextclade_command`